locales-src/extra_aliases.js

```diff
@@ -11,204 +11,224 @@
 module.exports = {
   de: {
     // German
-    "drehe dich nach links um %1 Grad": "turn @turnLeft %1 degrees",
-    "drehe dich nach rechts um %1 Grad": "turn @turnRight %1 degrees",
-    "Wenn die grüne Flagge angeklickt": "when @greenFlag clicked",
-    Ende: "end",
+    "drehe dich nach links um %1 Grad": "MOTION_TURNLEFT",
+    "drehe dich nach rechts um %1 Grad": "MOTION_TURNRIGHT",
+    "Wenn die grüne Flagge angeklickt": "EVENT_WHENFLAGCLICKED",
+    Ende: "scratchblocks:end",
```

Comment:
Do we need to make this end -> scratchblocks:end change anywhere else? `stringify` perhaps?

Reply:
Checking

Reply:
stringify currently uses English `end` and does not translate that.

Reply:
Ooh interesting. We should probably change that, I’m happy for that to be a separate PR though!